ui/src/pages/View.js

```diff
@@ -56,14 +158,148 @@ class View extends Component {
         );
   }
 
+  handleSearch(query) {
+    this.setState({isLoaded: false});
+    this.props.history.push({
+        pathname: `/${this.repoId}/results`,
+        search: '?query_name=' + encodeURIComponent(query),
+      });
+  }
+
+  renderField(labelMessage, value) {
+    return (
+      <div>
+        <span className='view-fieldname'>
+          <FormattedMessage {...labelMessage} />:
+        </span>
+        <span> </span>
```

Comment:
Curious: why a span here? Is it bad to use &nbsp; to force a space?

Reply:
I see, yeah, that was what I was suggesting.

Reply:
FYI Looks like this is one of the common ways to do that according to StackOverflow (https://stackoverflow.com/a/41142436/4339963).